examples/.interop/next-prisma-starter/package.json

```diff
@@ -15,11 +15,12 @@
     "dx:prisma-studio": "pnpm prisma-studio-sqlite",
     "dx": "run-p dx:* --print-label",
     "dev": "pnpm dx:next",
-    "build-sqlite": "run-s generate-sqlite migrate-sqlite && next build",
+    "prep-sqlite": "run-s generate-sqlite migrate-sqlite",
+    "build-sqlite": "next build",
```

Comment:
made this separation cause we can't cache prisma stuff - but want to cache the nextjs build